deepgram/transcriptions.go

```diff
@@ -30,6 +30,7 @@ type LiveTranscriptionOptions struct {
 	Numbers          bool     `json:"numbers" url:"numbers,omitempty" `
 	Numerals         bool     `json:"numerals" url:"numerals,omitempty" `
 	Profanity_filter bool     `json:"profanity_filter" url:"profanity_filter,omitempty" `
+	Smart_format     bool     `json:"smart_format" url:"smart_format,omitempty" `
```

Comment:
Could you move it in the list so that it comes after Search and before Tag (to keep alphabetical order of properties)? 

Thanks for the submission! Looking forward to getting it merged in.

Reply:
Yes, I didn't notice the alphabetical order - done!